greenplumpython/func.py

```diff
@@ -154,7 +156,7 @@ def apply(self, expand: bool = False, column_name: Optional[str] = None) -> Data
         )
 
     @property
-    def function(self) -> "_AbstractFunction":
+    def _function(self) -> "_AbstractFunction":
```

Comment:
Since we add `_` before the name, maybe we can remove this property and use `self._func` directly